docs/user_guide.rst

```diff
@@ -210,16 +212,43 @@ or subclasses of it such as :class:`.IQMFakeAdonis` that represent certain quant
     job.result().get_counts()
 
 
-Above, we use an :class:`.IQMFakeAdonis` instance to run a noisy simulation of ``circuit`` on a simulated 5-qubit Adonis chip.
-If you want to customize the noise model instead of using the default one provided by :class:`.IQMFakeAdonis`, you can create
-a copy of the fake Adonis instance with updated error profile:
+Above, we use an :class:`.IQMFakeAdonis` instance to run a noisy simulation of ``circuit`` on a simulated 5-qubit Adonis
+chip. If you want to customize the noise model instead of using the default one provided by :class:`.IQMFakeAdonis`, you
+can create a copy of the fake Adonis instance with updated error profile:
 
 .. code-block:: python
 
     error_profile = backend.error_profile
     error_profile.t1s['QB2'] = 30000.0  # Change T1 time of QB2 as example
     custom_fake_backend = backend.copy_with_error_profile(error_profile)
 
+Running a quantum circuit on a facade backend
+---------------------------------------------
+
+Circuits can be executed against a "simulated" environment: an IQM server that has no real quantum computer hardware.
+Results from such executions are random bits. This may be useful when developing and testing software integrations.
+
+Since Qiskit on IQM contains :class:`.IQMFakeAdonis`, it is possible to combine the "simulated" remote execution with
```

Comment:
`.IQMFacadeBackend` ?

Reply:
I meant to refer to `IQMFakeAdonis` that is described in the section above this. I.e., since `IQMFakeAdonis` exists, we also created `IQMFacadeBackend`.

Reply:
Updated this sentence.